src/plays/custom-dropdown-component/CustomDropdownComponent.js

```diff
@@ -0,0 +1,65 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import { useState } from 'react';
+import CustomDropdown from './components/CustomDropdown';
+import data from './data/countries.json';
+import './styles.css';
+
+function transformDataToAppropriateFormat() {
+  return data.map((country) => {
+    return {
+      ...country,
+      value: country.name.toLowerCase(),
+      label: country.name
+    };
+  });
+}
+
+// WARNING: Do not change the entry componenet name
+function CustomDropdownComponent(props) {
+  const [isMulti, setIsMulti] = useState(false);
+  const [isSearchable, setIsSearchable] = useState(false);
+  const [value, setValue] = useState(null);
+
+  return (
+    <div className="play-details">
+      <PlayHeader play={props} />
+      <div className="play-details-body">
```

Comment:
Add a unique class name along with `play-details-body` and use that unique style name to prefix all the styles in the CSS files.

Reply:
@atapas 
> Add a unique class name along with `play-details-body` and use that unique style name to prefix all the styles in the CSS files.


I've added a unique class name by the side of play-details-body and used that class on every stylesheet.

> @Jisan-mia
> 
> There are some issues when I select the multi check box
> 
> ![image](https://user-images.githubusercontent.com/3633137/242551767-5d6ff3dc-0725-47eb-9da2-c265a10df286.png)

Somehow, this error not showing on my side also I couldn't find where I coded `A.filter` on my project, although I double-checked and added optional chaining where filter method being used